addons/website_event_migration_layer/views/website_event.xml

```diff
@@ -0,0 +1,414 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<!-- =================================================== -->
+<!-- addons/website_event/views/event_templates_page.xml -->
+<!-- =================================================== -->
+<template id="navbar__ml" inherit_id="website_event.navbar" name="navbar__ml">
+    <xpath expr="//div" position="replace"/>
```

Comment:
Can we be a bit more specific about the divs? Even if it works, it seems weird to have the same selector twice